ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/ManageSubscriptionsView.kt

```diff
@@ -261,3 +270,12 @@ private fun NoActiveSubscriptionsViewNoDescription_Preview() {
         onAction = {},
     )
 }
+
+@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+internal fun PurchaseInformation.isSupportedPath(pathType: PathType): Boolean {
+    return if (isLifetime) {
+        pathType != CustomerCenterConfigData.HelpPath.PathType.CANCEL
+    } else {
+        true
+    }
+}
```

Comment:
I would have this directly as a function in `PurchaseInformation`? Even there it feels like `PurchaseInformation` should just be holding data and not doing any logic. Maybe we can have `supportedPaths` as a parameter of `ManageSubscriptionsView`? 

It could be part of `CustomerCenterState.Success` and be calculated at `loadCustomerCenter` after loading the `purchaseInformation` in the `viewModel`

That way we don't have to do filtering in the view (`screen.supportedPaths.filter { isSupportedPath(it.type) }`), and the filtering logic stays in the view model which is easier to test. Right now we are not testing that the filtering is actually happening.